azure/deploy-to-azure/azure-integration/CreateUIDefinition-managementgroups.json

```diff
@@ -275,12 +275,23 @@
                 "type": "Microsoft.Common.CheckBox",
                 "label": "Collect Custom Metrics",
                 "toolTip": "This option will enable custom metric collection from all Application Insights instances within the scope of this configuration. These custom metrics will appear in datadog under the namespace <b>application_insights.custom.&lt;METRIC_NAME&gt;</b>.<br> Note that <a target='_blank' href='https://learn.microsoft.com/en-us/azure/azure-monitor/app/standard-metrics/'>Standard</a> App Insights metrics are collected automatically and are available under the <b>azure.insights_components.*</b> namespace."
+              },
+              {
+                "name": "excluded_resource_providers",
+                "type": "Microsoft.Common.TextBox",
+                "label": "Excluded Resource Providers",
+                "defaultValue": "",
+                "toolTip": "Comma-separated list of Microsoft Resource Provider names to be excluded from metrics collection.",
```

Comment:
is a customer going to know what a resource provider is/is there any concern around case sensitivity here? an example of what we expect at the very least would be good, if not a more robust solution to validate that the customer had inputted only valid resource providers

Reply:
There is no concern around case sensitivity. the backend code is made to be case insensitive. It is a good idea to validate whether the users are sending accepted resource provider names. I'll include a PR in dogweb to make the validation.

Reply:
> There is no concern around case sensitivity.
Probably worth noting for users then

Reply:
included example